### PR TITLE
Yiwei/helper tests

### PR DESCRIPTION
**Pull request by zywkloo**
- Add a helper for  get_metrics_from_df
- update conda

### CODE REVIEW

File: src/helpers/get_metrics_from_df.py [hunk 1]
```diff
@@ -1,26 +1,34 @@
 from sklearn.metrics import mean_absolute_error, mean_squared_error, mean_absolute_percentage_error
 import pandas as pd
 
-
-def get_metrics_from_df(df, target_column: str, required_metrics: list = ["mae", "mse"]):
+def get_metrics_from_df(df, observation_column: str, required_metrics: list = ["mae", "mse"]):
     """
-    Calculate specified metrics for each forecast column in a DataFrame compared to a target column.
+    Calculate specified metrics for each forecast column in a DataFrame compared to an observation column.
```
Comment (SoloSynth1) [inline src/helpers/get_metrics_from_df.py:6]: What's the reason behind this name change?

File: src/helpers/get_metrics_from_df.py [hunk 2]
```diff
@@ -33,20 +41,21 @@ def get_metrics_from_df(df, target_column: str, required_metrics: list = ["mae",
...
-            mae_values.append(mae)
-        if require_mse:
-            mse = mean_squared_error(df[target_column], df[col])
-            mse_values.append(mse)
-        if require_mape:
-            mape = mean_absolute_percentage_error(df[target_column], df[col])
-            mape_values.append(mape)
+    # Calculate metrics for each forecast column compared to the observation column
+    for col in df.columns:
+        if col != observation_column:
```
Comment (SoloSynth1) [inline src/helpers/get_metrics_from_df.py:46]: Can we just do a `df[observation_column]` to remove this `for` loop?

**Review by SoloSynth1 — COMMENTED** [review]
@zywkloo Please see the comments. The tests look good to me.

File: src/helpers/get_metrics_from_df.py (hunk 1 above)
Comment (zywkloo) [reply src/helpers/get_metrics_from_df.py:6]: Because the original name is ["Observation"]

File: src/helpers/get_metrics_from_df.py (hunk 2 above)
Comment (SoloSynth1) [reply src/helpers/get_metrics_from_df.py:46]: Sorry, I misread. But I think it can still be simplified using the .drop() function

Comment (SoloSynth1) [reply src/helpers/get_metrics_from_df.py:46]: Actually forget this, not gonna copy the df to do the dropping

Comment (SoloSynth1) [reply src/helpers/get_metrics_from_df.py:46]: Resolved.

**Review by SoloSynth1 — APPROVED** [review]
LGTM. Merging to `main`.

**Pull request merged**